includes/admin/class-wc-rest-payments-webhook-controller.php

```diff
@@ -190,7 +190,7 @@ private function process_webhook_refund_updated( $event_body ) {
 					'code'   => '<code>',
 				]
 			),
-			wc_price( $amount / 100 ),
+			wc_price( $amount / 100, [ 'currency' => WC_Payments_Utils::get_order_original_currency( $order ) ] ),
```

Comment:
In this case, we can access the currency information in the event itself: https://stripe.com/docs/api/refunds/object - performance, and context friendly.

Reply:
Good idea! Here it is [abb8804](https://github.com/Automattic/woocommerce-payments/pull/1227/commits/abb8804021f924f8a7488f5d13cd5bfd3fde1b65).